code/game/machinery/doors/firedoor.dm

```diff
@@ -36,8 +36,8 @@
 		. += "<span class='notice'>It is open, but could be <b>pried</b> closed.</span>"
 	else if(!welded)
 		. += "<span class='notice'>It is closed, but could be <b>pried</b> open.</span>"
-		. += "<span class='notice'>Hold the door open by prying it with <i>left-click</i> and standing next to it.</span>"
-		. += "<span class='notice'>Prying by <i>right-clicking</i> the door will simply open it.</span>"
+		. += "<span class='notice'>Hold the firelock temporarily open by prying it with <i>left-click</i> and standing next to it.</span>"
+		. += "<span class='notice'>Prying by <i>right-clicking</i> the firelock will open it permanently.</span>"
```

Comment:
This is a bit of a mis-characterization, right click doesn't perma open it, it just leaves it at the mercy of the area based firelock timer

Reply:
i dunno if there is a better way to word it. here "permanently" mean "not temporarily". in the grand scheme of things nothing is permanent 🤔